swagger.yaml

```diff
@@ -3754,6 +3764,21 @@ definitions:
       console_url:
         type: string
 
+  disk-config-params:
+    type: object
+    required:
+      - 'id'
+      - 'path'
+    properties:
+      id:
+        type: string
+      path:
```

Comment:
Didn't we agree to remove this because it's easy to get path, given id (by looking at the inventory). Thus making it redundant?

Reply:
Done